.github/workflows/docs_website_publish.yaml

```diff
@@ -0,0 +1,59 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "website/**"
+
+env:
+  WORKING_DIR: ./website
+
+defaults:
+  run:
+    working-directory: ./website
+
+jobs:
+  build:
+    name: Build Docusaurus
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
```

Comment:
since we are using `packageManager` property on `package.json` we have to enable corepack before `actions/setup-node`

see https://github.com/pagopa/dx/blob/main/.github/workflows/js_code_review.yaml#L14